server/web/templates/proposals/submissions-list.handlebars

```diff
@@ -134,7 +137,28 @@
                                                 {{/if}}                                                 
                                                 <button type="button" class="btn btn-secondary" data-toggle="modal" data-target="#reviewersList-{{_id}}">
                                                     See Reviewers
-                                                </button>                                                
+                                                </button>
+                                                {{#and history.hasHistory hasFeedback}}
+                                                    <button class="btn btn-secondary dropdown-toggle" type="button" id="proposal-history" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                                        {{#if ../history.isOriginal}}
+                                                            Revised proposals
+                                                        {{else}}
+                                                            All proposals
+                                                        {{/if}}
+                                                    </button>
+                                                    <div class="dropdown-menu dropdown-menu-right" aria-labelledby="proposal-history">
+                                                        {{#unless ../history.isOriginal}}
+                                                            <a class="dropdown-item" href="/api/S3/getObject/{{../history.original.fileName}}">
+                                                                {{../history.original.fileName}} (Uploaded on {{../history.original.dateString}})
```

Comment:
Change history to revisedProposals see my comments here: https://github.com/hicsail/framingham-heart-study/pull/14/files#r1160430218